packages/core/src/components/fields/SchemaField.js

```diff
@@ -268,7 +270,7 @@ function SchemaFieldRender(props) {
       isMultiSelect(schema, rootSchema) ||
       isFilesArray(schema, uiSchema, rootSchema);
   }
-  if (schema.type === "object") {
+  if (schema.type === "object" || schema.type === undefined) {
```

Comment:
I think it would be better to say `schema.oneOf || schema.anyOf` -- would that work?